<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAuxDictionary</class>
 <widget class="QDialog" name="CAuxDictionary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dictionary</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="verticalSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="editWord"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnClear">
           <property name="text">
            <string/>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listWords"/>
       </item>
      </layout>
     </widget>
     <widget class="CGDTextBrowser" name="browser"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CGDTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>specwidgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editWord</tabstop>
  <tabstop>btnClear</tabstop>
  <tabstop>listWords</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>editWord</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
